MapboxVision/src/main/java/com/mapbox/vision/models/video/VideoTelemetry.kt

```diff
@@ -0,0 +1,11 @@
+package com.mapbox.vision.models.video
+
+import com.mapbox.vision.mobile.core.models.VideoClip
+
+internal data class VideoTelemetry(val startSeconds: Float, val endSeconds: Float)
```

Comment:
what is the difference between that class and `VideoStartStop`
can we use only one of them?

Reply:
nope, it's different objects, for different aims. From names: `VideoTelemetry` is about telemetry specific and `VideoStartStop` is common